src/_provisional.scss

```diff
@@ -286,6 +286,29 @@
 		}
 	}
 
+	/* Responsive table - part of design, not responsive part */
+	&.gc-table  {
+		caption {
+			text-align: left;
+		}
+		&.table-small-txt {
```

Comment:
Leverage the "small" CSS that is included in bootstrap. And then just add the variation you need for the `<tr>` or `<thead>`